producer-kafka/src/main/java/io/stargate/producer/kafka/KafkaCDCProducer.java

```diff
@@ -32,15 +31,21 @@
 import io.stargate.producer.kafka.schema.SchemaRegistryProvider;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
 import java.util.function.Consumer;
 import org.apache.avro.generic.GenericRecord;
 import org.apache.cassandra.stargate.db.DeleteEvent;
 import org.apache.cassandra.stargate.db.MutationEvent;
 import org.apache.cassandra.stargate.db.RowUpdateEvent;
 import org.apache.kafka.clients.producer.ProducerRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class KafkaCDCProducer extends SchemaAwareCDCProducer {
-
+  private static final Logger LOGGER = LoggerFactory.getLogger(KafkaCDCProducer.class);
+  private static final Executor KAFKA_INIT_EXECUTOR = Executors.newSingleThreadExecutor();
```

Comment:
Since this is a one-use thread pool, is it worth keeping it open after `init`? What do you think about using a simple thread that would exit after running the `init` method?

Reply:
good point, done